examples/demonstrations/stealthissound.scd

```diff
@@ -468,7 +468,7 @@ PmonoArtic(
 )
 
 
- //richer sequence with some heavier moments
+//richer sequence with some heavier moments
```

Comment:
In your note on **technique** you mentioned:

> 2. for multiline comments, using "// " prefix instead

So should lines with `//xxx` be changed to `// xxx`? (space added after `//`)

Reply:
I suppose the question stands for single-line comments as well.